Source/AliveLib/Scrab.cpp

```diff
@@ -227,6 +227,42 @@ void Scrab::VUpdate()
     vUpdate_4A3530();
 }
 
+void Scrab::vOn_TLV_Collision_4A4B80(Path_TLV* pTlv)
+{
+    for (; pTlv != nullptr; pTlv = sPath_dword_BB47C0->TLV_Get_At_4DB290(
```

Comment:
This loop is really strange, I think it should be a while(pTlv) { pTlv = sPath_dword_BB47C0->TLV_Get_At_4DB290; }